sui/src/sui_commands.rs

```diff
@@ -445,6 +463,7 @@ pub async fn make_server(
     consensus_committee: &ConsensusCommittee<Ed25519PublicKey>,
     consensus_store_path: &Path,
     consensus_parameters: &ConsensusParameters,
+    net_parameters: Option<Vec<(AuthorityName, String, u16)>>,
```

Comment:
There's an `AuthorityInfo` type here that contains all you need:
https://github.com/MystenLabs/sui/blob/7fec599121497ad7811ceec326e425c48cd1ff55/sui/src/config.rs#L40-L57
You could use it with an `impl From<AuthorityPrivateInfo> for AuthorityInfo` and eliminate this type signature along with ~25 lines of iterator calls.

Reply:
Oh I like that!

Reply:
In fact there is already a lovely `pub fn get_authority_infos(&self) -> Vec<AuthorityInfo>` that I will use